docs/using.rst

```diff
@@ -125,6 +125,59 @@ becomes:
     :doc:`Patches welcome! <contributing>`
 
 
+Hiding author name from change lines
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need clean output without author's names in changelog,
+then you can specify this preference using the ``:hide_author: True``
+argument, for example:
+
+    .. git_changelog::
+        :hide_author: 1
+
+.. _the man page: https://www.kernel.org/pub/software/scm/git/docs/git-rev-parse.html
```

Comment:
These lines appear to have been copy-pasted from elsewhere; can we remove them here (and below), please?

Reply:
Of course we can, done